src/renderers/dom/stack/client/wrappers/ReactDOMInput.js

```diff
@@ -122,6 +123,7 @@ var ReactDOMInput = {
       initialChecked: props.checked != null ? props.checked : props.defaultChecked,
       initialValue: props.value != null ? props.value : defaultValue,
       listeners: null,
+      onBlur: _handleBlur.bind(inst),
```

Comment:
It sort of is but we read event handlers from the original props now. All the custom logic is in ChangeEventPlugin and restoreControlledState.